### DISCUSSION TITLE
is there anything is place for persistent db url creation

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#115] [Ideas]
is there anything is place for persistent db url creation

**Comment by euzu** [comment]
i dont understand this question

**Comment by aniel300** [comment]
right now the db/ url creation structure uses something like this `http://xxx/m3u-stream/x/x/1,2,etc.` for url structute. what is this structure uses md5, everything after provider domain, or some other persistent and permeant way as way to streams always have the same reachable url even if u delete and re add the provider.

**Reply by euzu** [reply]
I cant follow you, i really having difficulties to follow your explanation.
What is a db/url structure?

Why should you delete and add the provider.

I really dont understand what you are asking?

**Reply by aniel300** [reply]
never mind. it seems it already does this/ what i was trying to explain. i just double checked/ realized.
